Puppetfile.tracking

```diff
@@ -10,8 +10,9 @@ mod 'simp-doc',
 moduledir 'src/assets'
 
 mod 'rubygem-rubygem_simp_cli',
+# RELEASE ME
   :git => 'https://github.com/simp/rubygem-simp-cli',
-  :tag => '4.0.4'
+  :ref => 'master'
```

Comment:
Are these supposed to have master as the ref?  There are several of them in the tracking file but not all.

Reply:
@jeannegreulich I only changed tag->ref for components that need to be released, i.e., components that have mission-impacting changes.  This gives us a better idea of how close we are to releasing SIMP 6.2.0.